### PR TITLE
Home and dashboard view

### PR DESCRIPTION
**Pull request by fadyyousry**

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -1,2 +1,6 @@
 class HomeController < ApplicationController
+  def index
+    @new_books = Book.last(10).reverse
+    @recommends = Book.left_joins(:sales).group(:id).order('COUNT(sales.id) DESC').limit(10)
```
Comment (Magdyx) [inline app/controllers/home_controller.rb:4]: add those to model

File: app/controllers/manager/media_controller.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
+module Manager
+  class MediaController < ApplicationController
+    layout 'admin'
+    load_and_authorize_resource
+
+    def index
+      @medium = Medium.first
+      if @medium.nil?
+        @medium = Medium.new
```
Comment (Magdyx) [inline app/controllers/manager/media_controller.rb:9]: use [this](https://apidock.com/rails/ActiveRecord/Relation/first_or_initialize)

File: app/models/ability.rb [hunk 3]
```diff
@@ -35,5 +36,8 @@ def admin
     can :manage, User
     cannot :destroy, User, id: @user.id
     can :read, Sale
+    can :read, Review
+    can :destroy, Review
```
Comment (Magdyx) [inline app/models/ability.rb:40]: can he destroy review? I think he can't

File: db/seeds.rb [hunk 4]
```diff
@@ -5,3 +5,7 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+Medium.create({address: "The BookStore Theme, 4th Store Beside that building, USA",
+    call: "800 1234 5678", email: "info@bookstore.com",
+     timing: "Mon - Fri: 7am - 10pm\n​Saturday: 8am - 10pm\nSunday: 8am - 11pm"})
```
Comment (Magdyx) [inline db/seeds.rb:11]: usually you should add rails db:seed to travis and github actions

Comment (fadyyousry) [reply db/seeds.rb:11]: done

**Pull request merged**